packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -113,12 +113,11 @@ export class FluidDataStoreRuntime
 		sharedObjectRegistry: ISharedObjectRegistry,
 		existing: boolean,
 	): FluidDataStoreRuntime {
-		return new FluidDataStoreRuntime(
-			context,
-			sharedObjectRegistry,
-			existing,
-			async (dataStoreRuntime) => dataStoreRuntime.entryPoint,
-		);
+		return new FluidDataStoreRuntime(context, sharedObjectRegistry, existing, () => {
+			throw new UsageError(
```

Comment:
Who is using this?  Usually we shouldn't just destroy something after deprecating it, the assumption is people are still using it and transitioning to the new thing.

And is it removed in next?

Reply:
+1. Also find it weird to we added this to `ContainerRuntime.load()`, it kind of breaks it silently...? Users won't know it's broken until they see an exception thrown at runtime for something that the method is not even asking of them. I think we might need to revert that?

Reply:
Looks like this is what we did in ContainerRuntime.load... let's just delete these ASAP 😅